client/index.html

```diff
@@ -556,6 +556,15 @@ <h1 class="title">Help</h1>
 
 						<div class="help-item">
 							<div class="subject">
+								<code>/ban user</code>
```

Comment:
All other helpers are `/foo nick`, is this one different on purpose? If it's to highlight that not only nicks can be passed, then I suggest `/ban target` and then in the description `<code>target</code> can be a nickname, or a...` (with a better wording than what my 5-yo English can come up with lol).